src/keystone/shim_test.mak

```diff
@@ -0,0 +1,106 @@
+#
+#    File: shim_test.mak
+
+ifndef SRC_DIR
+SRC_DIR=..
+endif
+ifndef INC_DIR
+INC_DIR=../../include
+endif
+ifndef OBJ_DIR
+OBJ_DIR=.
+endif
+ifndef EXE_DIR
+EXE_DIR=.
+endif
+
+#ifndef GOOGLE_INCLUDE
+#GOOGLE_INCLUDE=/usr/local/include/google
+#endif
+
+LOCAL_LIB=/usr/local/lib
+
+ifndef TARGET_MACHINE_TYPE
+TARGET_MACHINE_TYPE= x64
+endif
+
+S= $(SRC_DIR)/keystone
+O= $(OBJ_DIR)
+I= $(INC_DIR)
+CL=..
+
+INCLUDE=-I $(I) -I/usr/local/opt/openssl@1.1/include/ -I .
+
+ifdef ENABLE_SEV
+CFLAGS=$(INCLUDE) -O3 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -D SEV_SNP -Wno-deprecated-declarations
+CFLAGS1=$(INCLUDE) -O1 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -D SEV_SNP -Wno-deprecated-declarations
+else
+CFLAGS=$(INCLUDE) -O3 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -Wno-deprecated-declarations
+CFLAGS1=$(INCLUDE) -O1 -g -Wall -std=c++11 -Wno-unused-variable -D X64 -Wno-deprecated-declarations
+endif
+
+CC=g++
+LINK=g++
+
+# Point this to the right place, if you have to, based on your machine's install:
+# PROTO=/usr/local/bin/protoc
+PROTO=protoc
+AR=ar
+#export LD_LIBRARY_PATH=/usr/local/lib
+LDFLAGS= -L $(LOCAL_LIB) -lprotobuf -lgtest -lgflags -lpthread -L/usr/local/opt/openssl@1.1/lib/ -lcrypto -lssl
+
+dobj = $(O)/certifier.pb.o $(O)/certifier.o $(O)/certifier_proofs.o        \
+       $(O)/support.o $(O)/application_enclave.o $(O)/simulated_enclave.o  \
+       $(O)/cc_helpers.o $(O)/cc_useful.o $(O)/keystone_shim.o $(O)/keystone_test.o
+
+all:	$(CL)/keystone_test.exe
+clean:
+	@echo "removing object files"
+	rm -rf $(O)/*.o
+	@echo "removing executable files"
+	rm -rf $(CL)/keystone_test.exe
```

Comment:
This `*.exe` is being built in current dir, `keystone/keystone_test.exe`.

Shouldn't `clean` rm from current dir rather than from `$(CL)` ???

Reply:
And, as exe generates the 2 `*.bin` files, can we have `clean` rm those too?

That way, we are sure to be starting from a clean slate w/o older .bin files hanging around.